### DISCUSSION TITLE
Mouse (sometimes) reacts to 'half'-scroll events

### DISCUSSION THREAD

**Discussion by AlexanderSlesarenko** [linearmouse/linearmouse#580] [General]
Hello, I use app with mac os 12.6.1.
Logitech mouse m90.
Everything is good except one moment.
Sometimes when i touch scroll wheel a little - so it moves just a little - without click sound - app reacts to this event and scrolls the page.
It's quite annoying - there is not such behaviour when i use this mouse with windows.

**Comment by lujjjh** [comment]
That's quite strange. If the system is responding to the scroll event, it means that the scroll wheel is indeed sending out the scroll event. If it cannot be reproduced in Windows, it suggests that Windows is filtering out certain scroll events based on certain rules.

I've tried several mice, but I can't reproduce this issue. If you can consistently reproduce it and export the logs, it might be helpful.
